doc/README.md

```diff
@@ -192,7 +192,10 @@ the graph, controlled by the *nodecount* option.
 
 * **Node Color**:
   * large positive cum values are red.
-  * large negative cum values are green.
+  * large negative cum values are green; negative values are most likely to
+    appear during profile comparison, see [this section]
+    (https://github.com/google/pprof/tree/master/doc#comparing-profiles) for
```

Comment:
I think the newline broke the link (see https://github.com/google/pprof/tree/nolanmar511-patch-1/doc).

And relative links like `[this section](#comparing-profiles)` would work in case this doc was branched.